arch/arm/src/nrf52/nrf52_sdc.c

```diff
@@ -321,21 +279,14 @@ static void on_hci(void)
           struct bt_hci_acl_hdr_s *hdr =
               (struct bt_hci_acl_hdr_s *)g_sdc_dev.msg_buffer;
 
-          wlinfo("received HCI ACL from softdevice (handle: %d)\n",
-                 hdr->handle);
-
```

Comment:
why remove this? it is useful for debugging

Reply:
Oops, I restore the debug info here